src/conspiracies/extract_ents_heads/functions.py

```diff
@@ -0,0 +1,77 @@
+import spacy
+from collections import Counter
+
+
+def get_headword(noun_phrases: list) -> list:
+  """Extracts headwords with their entity labels from noun phrases
+   Args:
+       noun_phrases (list): A list of lists with strings (noun phrases) 
+   Returns:
+       headwords (list): A list of strings containing headwords and their entity types
+  """ 
+
+  headwords = []
+
+  for phrase in noun_phrases:
+    for span in phrase:
+      for word in span:
+        if word.head.pos_ == "PROPN" or word.head.pos_ == "NOUN" or word.head.pos_ == "PRON":
```

Comment:
much simpler to write:

```suggestion
        if word.head.pos_ in {"PROPN", "NOUN", "PRON"}:
```

Reply:
Should this be an argument to the function instead:

i.e. somehting like
```
pos_to_keep = {"PROPN", "NOUN", "PRON"},

...
if word.head.pos_ in pos_to_keep:
```